camel/storages/vectordb_storages/milvusDB_v2.py

```diff
@@ -0,0 +1,99 @@
+from pymilvus import MilvusClient, Collection
```

Comment:
better to move the import into class

Reply:
I have moved it into the function, Thanks